docs/source/posts/2022/2022-07-20-week-7-praneeth.rst

```diff
@@ -0,0 +1,53 @@
+===========================================================
+Week 7 - Working on Rotation PR and Trying Freehand Drawing
+===========================================================
+
+.. post:: July 20 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+I continued PR `#623`_ and fixed the displacement of the shape from its original position when applying rotation. This was happening because all the results were type casted to ``int`` due to which some positions got rounded off. As we call this function continuously, each time the round-off would happen, the shape would get displaced.
```

Comment:
Missing a comma after `casted to int`. 
Should be `cast`. I'd also suggest rephrasing the sentence for more clarity.